### PR TITLE
docs: test translations locally

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

The question has come up a couple of times in chat, so I've documented the process for enabling a new language in the config. This will allow contributors to test their translations locally if they wish, though downloading the translations from Crowdin appears to be only available to staff.

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - **Translation Contribution**
   - [Work on translating resources](how-to-translate-files.md)
   - [Work on proofreading translations](how-to-proofread-files.md)
+  - [Test translations locally](how-to-test-translations-locally.md)
```
Comment (raisedadead) [inline docs/_sidebar.md:14]: Move this to the optional section in the sidebar.

File: docs/how-to-test-translations-locally.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    searchPage: 'https://chinese.freecodecamp.org/news/search'
+  },
+  dothraki: {
+    name: 'news',
+    searchPage: 'https://www.freecodecamp.org/news/search/'
+  },
+};
+```
+
+Finally, you will need to tell the client which certifications are translated, and which are still in English. Open the `utils/is-audited.js` file. Within the `auditedCerts`, add a new key with your language's `availableLangs` value. Assign the value of that key to an array containing the *dashed names* for the certifications that have been translated. Refer to the existing data for those dashed names.
```
Comment (raisedadead) [inline docs/how-to-test-translations-locally.md:83]:
```suggestion
Next, you will need to tell the client which certifications are translated, and which are still in English. Open the `utils/is-audited.js` file. Within the `auditedCerts`, add a new key with your language's `availableLangs` value. Assign the value of that key to an array containing the *dashed names* for the certifications that have been translated. Refer to the existing data for those dashed names.
```

There is use of Finally in paragraph after this.

**Review by raisedadead — CHANGES_REQUESTED** [review]
Some nit-picks.

**Review by raisedadead — DISMISSED** [review]

File: docs/how-to-test-translations-locally.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+If you would like to test your translations on a local instance of the freeCodeCamp `/learn` site, first ensure you have [set up the codebase](how-to-setup-freecodecamp-locally.md).
+
+## Enabling a Language
+
+There are a few steps to take in order to allow the codebase to build in your desired language.
+
+First, visit the `config/i18n/all-langs.js` file to add the language to the available languages list and configure the values. There are four objects here.
+
+- `availableLangs`: For both the `client` and `curriculm` arrays, add the text name of the language. This is the value that will be used in the `.env` file later.
```
Comment (ShaunSHamilton) [inline docs/how-to-test-translations-locally.md:14]:
```suggestion
- `availableLangs`: For both the `client` and `curriculum` arrays, add the text name of the language. This is the value that will be used in the `.env` file later.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Teeny, tiny thing...

**Review by raisedadead — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM, and will be a useful resource to direct contributors to 🎉

File: docs/how-to-test-translations-locally.md (hunk 3 above)
Comment (nhcarrigan) [reply docs/how-to-test-translations-locally.md]: I promise I'll learn how to spell one day. 🙂

**Pull request merged**